docs/platform/nodebalancer/force-all-connections-on-ssl-with-nodebalancers.md

```diff
@@ -0,0 +1,370 @@
+
+    An example of what this file will look like can be found below (yours will be unique):
+
+        -----BEGIN PRIVATE KEY-----
+        MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDhqR/0T9uiZY0s
+        tQ27f0es1OVf3FkHfAjCsM1G2uMtGadRAFQN7lsa+W0W0tzfeJGSk4tqDanOPeJU
+        3jm/Udxy18DSl2xYKhJAF8YL50UPN6Yih1p+XPIEBqKjAEn+p7BgH/R7vehZQh+z
+        ................................................................
+        ZegL7vs+//1jBJy3h/dcFNl7Pgiv7UlsxXhFw7aZAoGATUW7pWP1osmMFTshW6xo
+        qSdRwPnDjilI9r9SEEwlcssGIYReIUfRuXnaaPLudKDeZ3GZ0kjXfbsWFOnwLyAE
+        pNaMF47bTIFES5Ytqhp4rEYe5Hxn2VePrGJp/j7tAIzTXeV3IC6ICkEueKpKS2Dl
+        xHtX/Rl40JHhTCiGK5eIR3E=
+        -----END PRIVATE KEY----
+
+### Submit your CSR to a Certificate Authority
+
+4.  You may now submit the contents of this file to a commercial SSL provider (Certificate Authority) for signing. Typically, they will provide you with a field to paste the contents of your CSR file into, somewhere on their website. The following is a list of well known commercial SSL vendors, provided for your convenience:
+
+    - [Verisign](https://www.verisign.com/)
+    - [Thawte](https://www.thawte.com/)
+    - [Globalsign](https://www.globalsign.com/en/)
+    - [Comodo](https://www.comodo.com/)
+
+      {:.note}
+     > 
+     > There are many vendors, including some listed above, that provide free SSL certificates. However, free SSL certificates typically have shorter expiration dates, and less features.
+
+5.  Once this request has been submitted, you will receive a commercially signed SSL certificate file, which will look similar to the following (yours will be unique):
+
+        -----BEGIN CERTIFICATE-----
+        MIIFSzCCBDOgAwIBAgIQVjCXC0bF9U8FypJOnL9cuDANBgkqhkiG9w0BAQsFADCB
+        kDELMAkGA1UEBhMCR0IxGzAZBgNVBAgTEkdyZWF0ZXIgTWFuY2hlc3RlcjEQMA4G
+        A1UEBxMHU2FsZm9yZDEaMBgGA1UEChMRQ09NT0RPIENBIExpbWl0ZWQxNjA0BgNV
+        ................................................................
+        RWieiEDSZqbtZRPBaGooDJ5QbdHqUanvoVzf1aB1S5RrJB5qH/UG6WbTZ07rFfsn
+        age7UPo4ZwheAtpO2mhcYypBG1zln4cvxVBAcrnaa1GWwKjgwXUr5k2Pv7BXWEex
+        ncHG3hwHHwhiEz6ukC2mqxA+D3KILiywgHgWcumnpeCEUQgDzy0Fz2Ip/kR/1Fkv
+        DCQzME2NkT1ZdW8fdz+Y
+        -----END CERTIFICATE-----
+
+6.  Save this file as `/etc/ssl/localcerts/www.mydomain.com.crt`. Execute the following command to protect the signed certificate:
+
+        chmod 400 /etc/ssl/localcerts/www.mydomain.com.crt
+
+### Preparing a Chained SSL Certificate
+
+7.  However, you may receive several files ending with `.crt`. If this is the case, then they are collectively refereed to as a `chained SSL certificate` and must be concatenated into one file in order to provide full support with most browsers. The following example uses a chained SSL certificate that was signed by Comodo, but other vendors are perfectly reputable as well. Enter the following command to do this:
```

Comment:
Please adjust this explanation to explain the general order in which you would concatenate these certs, from Authority to domain-specific.